android/src/main/java/ie/gov/tracing/Tracing.kt

```diff
@@ -269,10 +291,36 @@ class Tracing {
         }
 
         @JvmStatic
-        fun stop() {
+        fun pause(promise: Promise?) {
             try {
                 setNewStatus(STATUS_STOPPING)
+                SharedPrefs.setBoolean("servicePaused", true, context)
```

Comment:
Should be done in the success/failure blocks?

Reply:
We miss state changes as paused if we do it in success block